### ISSUE TITLE
Mutations missing when multiple nodes

### ISSUE THREAD

**Issue by legatoslide** [couchbase/couchbase-kafka-connector#13]
Hello,

there are missing mutations when several nodes in the cluster.
The mutations are received from 1 DCPConnection instance in the Core IO. But potantially several DCPConnection can be instanciated and the DCPRequest they receive are not transmitted.

Regards,
David

**Comment by tk6022** [comment]
@legatoslide Is you connector configured to point at all nodes in your cluster and you are still seeing this?

**Comment by legatoslide** [comment]
@tk6022 Maybe I am missing something but I do not see how it can work as expected. I have made a dirty fix to gather DCPRequest from all DCPConnection instances and it seems to solve the issue.

**Comment by avsej** [comment]
Guys, thank you for report. I've reproduced it and will fix it soon

**Comment by tk6022** [comment]
@avsej Can you please comment on when this and the other open issues are likely to be addressed in a release please, and when the next release is expected.

Thanks

**Comment by WillGardella** [comment]
I synced up with @avsej last week. It looks like he's tracked down some
issues in the JVM Core library that will be worked on and released
there. He's working on getting the exact issue list together for that
release.

The dependent connectors, like Kafka, will get updated after the JVM Core
lib maintenance release around early March. We've discussed bumping up the
next scheduled maintenance release of the Kafka Connector to get those
fixes out earlier. I can't say exactly what will get fixed at this point or
the exact release data but we're working on it.
